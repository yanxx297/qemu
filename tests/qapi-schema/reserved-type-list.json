# Potential C name collision
# FIXME - This parses and compiles on its own, but prevents the user from
# creating a type named 'Foo' and using ['Foo'] for an array.  We should
# reject the use of any type names ending in 'List'.
{ 'struct': 'FooList', 'data': { 's': 'str' } }
