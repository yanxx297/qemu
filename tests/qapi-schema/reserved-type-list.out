object :empty
object FooList
    member s: str optional=False
